docs/source/yatai_wordlist.txt

```diff
@@ -0,0 +1,164 @@
+BentoML
+BentoService
+Colab
+Conda
+Dataframe
+Dockerfile
+ECS
+EvalML
+Evalml
+Floydhub
+HTTP
+Http
+Init
+KNative
+Knative
+Kops
+Kubeflow
+Kubernetes
+Logloss
+MLlib
+Mesos
+Nginx
+NumpyNdarray
+OpenAPI
+Postgres
+Protobuf
+Pydantic
+Rossmann
+SLO
+Sagemaker
+Scikit
+Seldon
+Statsmodels
+Tensorflow
+Yatai
+YataiService
+agaricus
+ai
+api
+args
+autoscale
+aws
+backend
+bento
+bentofile
+bentoignore
+bentoml
+bentos
+catboost
+cbt
+clf
+cli
+conda
+config
+configs
+coreml
+coremltools
+cpu
+csv
+ctx
+dataframe
+dataset
+detectron
+dev
+dockerization
+docstring
+dtest
+dtrain
+dtype
+dtypes
+dtypes
+easyocr
+ecd
+ecr
+evalml
+fastai
+gRPC
+gbdt
+gbm
+gitignore
+gpt
+gpu
+grafana
+healthz
+heroku
+hostname
+hparams
+http
+huggingface
+inp
+inplace
+istio
+jaeger
+joblib
+json
+keras
+kubectl
+kubernetes
+kunlun
+kwargs
+lgb
+lightgbm
+livenessProbe
+localhost
+microbatch
+minikube
+mlir
+mxnet
+namespace
+ndarray
+ngrok
+numpy
+onnx
+onnxruntime
+openapi
+openzipkin
+otlp
+outcoming
+param
+picklable_model
+pre
+pretrained
+prometheus
+pybind
+pycaret
+pypi
+pypirc
+pytorch
+quickstart
+readyinessProbe
+recog
+resnet
+roadmap
+roberta
+rollout
+runtime
+sagemaker
+scikit
+serverless
+sklean
+sklearn
+softmax
+spacy
+standardlize
+statsmodels
+subclassing
+svc
+tensorflow
+tf
+tfhub
+tokenizer
+torchvision
+trf
+txt
+unpickle
+url
+utf
+util
+vocab
+xgb
+xgboost
+yaml
+yatai
```

Comment:
Should we ensure all occurrences of Yatai is capitalized?

Reply:
@ssheng If yatai-deployment cannot be capitalized, it will be asymmetrical when put together with Yatai